var/spack/repos/builtin/packages/codecov/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import platform
+
+from spack.package import *
+
+
+class Codecov(Package):
+    """Codecov uploads coverage reports to Codecov for processing."""
+
+    homepage = "https://codecov.io"
+    _url_base = "https://github.com/codecov/uploader/releases/download/v0.4.1/codecov"
```

Comment:
Prob a better idea to use a `url_for_version` since this won't work if you ever add a new version. See #36830 for a recent example I did.

Reply:
I'll implement that approach tomorrow or likely Saturday.

Reply:
Implemented.